griptape/drivers/vector/azure_mongodb_vector_store_driver.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional
+from attr import define, field, Factory
+from griptape.drivers import BaseVectorStoreDriver, MongoDbAtlasVectorStoreDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from pymongo import MongoClient
+    from pymongo.collection import Collection
+
+
+@define
+class AzureMongoDbvCoreVectorStoreDriver(MongoDbAtlasVectorStoreDriver):
```

Comment:
AzureMongoDb**v**CoreVectorStoreDriver

Extra `v`?

Reply:
the API is called `vCore`. any name suggestions?

Reply:
updated to `AzureMongoDbVectorStoreDriver`